splitting.go

```diff
@@ -21,6 +21,7 @@ const DefaultBlockSize int64 = 1024 * 256
 type Splitter interface {
 	Reader() io.Reader
 	NextBytes() ([]byte, error)
+	Chunksize() uint64
```

Comment:
Sorry, can we make this `ChunkSize()` ? matches `NextBytes()` naming

Reply:
k.